test/bats/callback.bats

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bats
+
+load "helpers/setup-and-teardown"
+load "helpers/ln"
+
+setup_file() {
+  clear_cache
+
+  bitcoind_init
+  start_trigger
+  start_ws_server
+  start_server
+  start_exporter
+  start_callback
+
+  lnds_init
+}
+
+teardown_file() {
+  stop_trigger
+  stop_server
+  stop_ws_server
+  stop_exporter
+  stop_subscriber
+  stop_callback
+}
+
+setup() {
+  reset_redis
+}
+
+teardown() {
+  if [[ "$(balance_for_check)" != 0 ]]; then
+    fail "Error: balance_for_check failed"
+  fi
+
+}
+
+@test "setup callback endpoints" {
```

Comment:
We have been using the convention to prefix the test name with the filename `callback: setup callback endpoints`.